packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/configure.yml

```diff
@@ -66,7 +66,7 @@
 
 - name: Verify fapolicyd.rules file
```

Comment:
Maybe use `Verify fapolicyd/rules.d directory` for the task name.

Reply:
Changed